react/week3/todo-app/src/Components/Todo.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import Border from "./Border";
+import PropTypes from "prop-types";
+function Todo({ toDo, deleteTodo, updateTodo }) {
+  const [status, setStatus] = useState(false);
+  const [isEditing, setEditState] = useState(false);
```

Comment:
```suggestion
  const [isEditing, setIsEditing] = useState(false);
```
https://reactjs.org/docs/hooks-state.html

Reply:
Thanks for your feedbacks, Time and good patience.
sorry for making a lot of naming error. 
I will take care from next time :heart_eyes: